src/web/mjs/engine/Enums.mjs

```diff
@@ -12,4 +12,8 @@ export default class Enums {
             onSelectManga:            'onSelectManga'
         };
     }
+
+    static get NovelColorProfiles() {
```

Comment:
Name is misleading, it provides the currently selected color profile and not a list of color profiles

Reply:
you were the one that made the name
https://github.com/manga-download/hakuneko/pull/2975#discussion_r565007963

Reply:
il swap em anyways